packages/react/src/components/Menu/MenuItem.tsx

```diff
@@ -255,8 +256,8 @@ class MenuItem extends AutoControlledComponent<WithAsProp<MenuItemProps>, MenuIt
         <>
           <Ref innerRef={this.menuRef}>
             <Popper
-              align={vertical ? 'top' : 'start'}
-              position={vertical ? 'after' : 'below'}
+              align={vertical ? 'top' : rtl ? 'end' : 'start'}
```

Comment:
Stardust noobie  question here for @codepretty, @levithomason , etc.: would this be better done within `Popper`? Making Popper itself respect RTL settings would eliminate this work on the consumer's end, which may otherwise be forgotten.

Reply:
I'm not sure. Popper seems to be a bit more generic - it depends on the element using it to feed it position information, and the positioning of Menu is affected by RTL possibly differently than other items using Popper. @levithomason ?
https://github.com/stardust-ui/react/blob/master/packages/react/src/lib/positioner/Popper.tsx

Reply:
@bcalvery, i think you have a good solution. i think it probably makes sense to keep this rtl logic where you have it unless we know how popper will be used in all cases, it might not make sense to prematurely move this logic to popper.